test/Edge-SDV/modules/edgecore/eventbus/eventbus_test.go

```diff
@@ -0,0 +1,214 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"database/sql"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"time"
+
+	MQTT "github.com/eclipse/paho.mqtt.golang"
+	"github.com/kubeedge/kubeedge/test/Edge-SDV/utils/common"
+	_ "github.com/mattn/go-sqlite3"
```

Comment:
seperate kubeedge and vendor packages with a blank line.
https://github.com/kubeedge/kubeedge/blob/master/pkg/metamanager/msg_processor_test.go see for example